gradle/libs.versions.toml

```diff
@@ -1,18 +1,18 @@
 [versions]
-kotlin = "1.8.21"
+kotlin = "1.9.0"
```

Comment:
I think 1.9.10 would avoid some potential Xcode 15 issues

Reply:
It definitely would, but decided to reduce extra changes as part of this merge